src/comonents/InputForm.jsx

```diff
@@ -1,11 +1,21 @@
-import React from 'react'
+import React, { useContext } from "react";
+import ButtonPage from "./ButtonPage";
+import { MyNoteContext } from "../context/NoteState";
 
 function InputForm() {
+  const { todo, handleChange } = useContext(MyNoteContext);
   return (
-    <div>
-      
+    <div style={{ width: "100%", display: "flex" }}>
+      <input
+        style={{ width: "100%" }}
+        type="text"
+        placeholder="Enter your text"
+        value={todo}
+        onChange={handleChange}
+      />
+      <ButtonPage />
```

Comment:
use appropriate name for component, this is not page,
use like button.

Reply:
Resolved sir